examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,214 @@
+
+.. code-block:: ini
+    :caption: **conanfile.txt**
+
+    [requires]
+    fmt/9.1.0
+
+    [generators]
+    AutotoolsToolchain
+    PkgConfigDeps
+
+In this case, we will use :ref:`PkgConfigDeps<conan_tools_gnu_pkgconfigdeps>` to generate information about where the **fmt** library
+files are installed thanks to the `*.pc` files and :ref:`AutotoolsToolchain<conan_tools_gnu_autotoolstoolchain>` to pass build information
+to *autotools* using a `conanbuild[.sh|.bat]` file that describes the compilation environment.
+
+We will use Conan to install **fmt** and generate the files that Autotools needs to find this library and build our project.
+
+
+Building on Linux and macOS
+---------------------------
+
+First, we should install the requirements. For this example we didn't consider your profile, but ``fmt`` is higly compatible with many different configurations.
+
+.. code-block:: shell
+
+    conan install . --build=missing
+
+Now we are ready to build and run our **string_formatter** app:
+
+.. code-block:: shell
+
+    source conanbuild.sh
+
+The ``conanbuild.sh`` generated by the :ref:`VirtualBuildEnv<conan_tools_env_virtualbuildenv>` will fill all environment
+variables according to your profile, plus, Autotools, fmt .pc files and ``PKG_CONFIG_PATH`` as environment variable.
+
+.. code-block:: shell
+
+    aclocal
+    automake --add-missing
+    autoconf
+    ./configure
+    make
+
+Now we generate the ``configure`` and ``Makefile``, then we build our project by running the ``make`` command.
+
+As final result, we will have the binary ``string_formatter`` installed in our example folder.
+
+.. code-block:: shell
+
+    ./string_formatter
+    Conan - The C++ Package Manager!
+
+
+Building on Windows
+-------------------
+
+To build on Windows, we will use a pre-defined profile as an example to give a better view of our current configuration.
+Also, we will be running on the default command prompt, it can be invoked by ``cmd.exe``.
+
+.. code-block:: ini
+    :caption: **profiles/msys2**
+
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.exception=seh
+    compiler.libcxx=libstdc++
+    compiler.threads=posix
+    compiler.version=11
+    os=Windows
+    [options]
+    */*:shared=True
+    [conf]
+    tools.cmake.cmaketoolchain:generator=MSYS Makefiles
+    tools.microsoft.bash:subsystem=msys2
+    tools.microsoft.bash:active=True
+    [tool_requires]
+    msys2/cci.latest
+    autoconf/2.71
+    automake/1.16.5
+    pkgconf/1.9.3
+    [buildenv]
+    CC=gcc
+    CXX=g++
+
+With this profile, we will build on Windows, with MSYS2 support and GCC 11. Some build requirements will be needed too.
+
+As the first step, we will install MSYS2 Conan package, to be able to run bash inside of our command prompt.
+
+.. code-block:: batch
+
+    conan install -r conancenter --requires=msys2/cci.latest -g VirtualBuildEnv
+    conanbuild.bat
+    set MSYSTEM=MSYS
```

Comment:
why not put it into profile?

Reply:
Removed Windows section for simplification.